### ISSUE TITLE
porno800.com

### ISSUE THREAD

**Issue by Rygle** [Clefspeare13/pornhosts#296]
I believe this domain is an Adult(-related) domain --> that have to be blocked as..

## Domains
Only add the needed one See also #12

```python
porno800.com
sex.porno800.com
www.sex.porno800.com
```

## Content sources

(Please see #35)

```python


```

## External sources (reference)

```python


```

## Screenshots

<details><Summary>Screenshot required</summary>

![](https://i.ibb.co/tm6xbKn/Screenshot-Firefox-Beta-20201010-080753.png)

</details>

## External Info
- 
- 
- 

### All Submissions:
- [x] Have you checked to ensure there aren't other open [Merge Requests (MR)](../merge_requests) or [Issues](../issues) for the same update/change?
- [x] Added a Screenshot for prove of Adult contents

### Todo
- [x] Added to [Source file](submit_here/hosts.txt)?
- [x] Added to the RPZ zone [adult.mypdns.cloud](https://www.mypdns.org/w/rpzlist/#adult-mypdns-cloud) (@spirillen)

**Comment by spirillen** [comment]
Hey @Rygle 

What DNS servers are you using?? for this domain is actually non existing....

```shell
whois sexporno800.com
No match for domain "SEXPORNO800.COM".
>>> Last update of whois database: 2020-10-10T15:09:37Z <<<
```

**Comment by Rygle** [comment]
Corrected a typo. Should have stuck to copy and paste! 😉

**Comment by spirillen** [comment]
Even, it is a very weird domain setup....

```python
sex.porno800.com
www.sex.porno800.com
```

Using `www` as fourth level domain for a website.

I'll add it when the test => distribution of yesterdays work is published

**Comment by spirillen** [comment]
Closed with 2c9a198
